mlflow/pyfunc/__init__.py

```diff
@@ -415,6 +415,43 @@ def predict(self, data: PyFuncInput) -> PyFuncOutput:
     def unwrap_python_model(self):
         """
         Unwrap the underlying Python model object.
+
+        This method is useful for accessing custom model functions, while still being able to
+        leverage the MLflow designed workflow through the `predict()` method.
+
+        .. code-block:: python
+            :caption: Example
+
+            # define a custom model
+            class MyModel(pyfunc.PythonModel):
+                def predict(self, context, model_input):
+                    self.my_custom_function(model_input)
+
+                def my_custom_function(self, model_input):
+                    # do something with the model input
```

Comment:
This is an invalid syntax. Let's return something.

```suggestion
                def my_custom_function(self, model_input):
                    # do something with the model input
                    return 0
```